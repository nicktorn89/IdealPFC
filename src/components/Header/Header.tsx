import React from 'react';
import { Navbar } from 'react-bulma-components/full';
import { HeaderProps } from './types';

const HeaderComponent: React.FC<HeaderProps> = ({}) => {

  return (
    <Navbar>
      <Navbar.Brand>
        <Navbar.Item>IdealPFC</Navbar.Item>
      </Navbar.Brand>

      <Navbar.Container position='end'>
        <Navbar.Item>Toggle in night mode</Navbar.Item>
      </Navbar.Container>
    </Navbar>
  );
};

export default HeaderComponent;
