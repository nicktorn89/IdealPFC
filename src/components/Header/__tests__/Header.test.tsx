import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { default as Header } from '../Header';

describe('<Header />', () => {
  const HeaderComponent = shallow(<Header />);
  const HeaderComponentToJson = shallowToJson(HeaderComponent);

  it('should be equal to snapshot', () => {
    expect(HeaderComponentToJson).toMatchSnapshot();
  });
});
