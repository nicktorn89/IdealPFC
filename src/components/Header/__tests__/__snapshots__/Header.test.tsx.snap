// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should be equal to snapshot 1`] = `
<t
  active={false}
  className=""
  color={null}
  fixed={null}
  renderAs="nav"
  style={Object {}}
  transparent={false}
>
  <c
    className=""
    style={Object {}}
  >
    <c
      active={false}
      className=""
      dropdown={false}
      dropdownUp={false}
      hoverable={false}
      renderAs="a"
      style={Object {}}
    >
      IdealPFC
    </c>
  </c>
  <c
    className=""
    position="end"
    renderAs="div"
    style={Object {}}
  >
    <c
      active={false}
      className=""
      dropdown={false}
      dropdownUp={false}
      hoverable={false}
      renderAs="a"
      style={Object {}}
    >
      Toggle in night mode
    </c>
  </c>
</t>
`;
